samples/java_springboot/14.nlp-with-dispatch/src/main/java/com/microsoft/bot/sample/nlpwithdispatch/DispatchBot.java

```diff
@@ -17,9 +17,6 @@
 import com.microsoft.bot.builder.RecognizerResult;
 import com.microsoft.bot.builder.TurnContext;
 import com.microsoft.bot.builder.RecognizerResult.NamedIntentScore;
-import com.microsoft.bot.sample.nlpwithdispatch.BotServices;
-import com.microsoft.bot.sample.nlpwithdispatch.Intent;
-import com.microsoft.bot.sample.nlpwithdispatch.PredictionResult;
```

Comment:
Is it correct to remove these? They are used in this file 🤔

Reply:
That's weird, I didn't manually remove them. No idea what happened.
However, the sample did work as it was, so it was either after testing it or they are not actually used.
I will validate this and revert it if necessary.

Reply:
Are you sure we need those imports? All those classes are inside the same package, so they are within the scope and reachable

Reply:
Now I remember what happened. The bot class was in a different folder, therefore needed those imports. After it was moved to the root folder, those imports were not needed anymore.